packages/fields/src/fields/featured-image/featured-image-view.tsx

```diff
@@ -30,6 +32,17 @@ export const FeaturedImageView = ( {
 				className="fields-controls__featured-image-image"
 				src={ url }
 				alt=""
+				srcSet={
```

Comment:
In the Edit function for the FeaturedImage field we could also use this technique to prevent the full image being requested for such small needs [24px](https://github.com/WordPress/gutenberg/blob/dfcc42d6103e8d810f89f284dfd35389723e913b/packages/fields/src/fields/featured-image/featured-image-edit.tsx#L74) (note the quick edit button for media):

<img width="1149" height="538" alt="Screenshot 2025-07-29 at 20 18 30" src="https://github.com/user-attachments/assets/d309608c-f013-4ac2-ad68-9cc9468985c8" />